src/Backend/Core/Layout/Templates/navigation.html.twig

```diff
@@ -13,7 +13,7 @@
   <a href="#" class="arrow-right js-nav-next"></a>
   <a href="#" class="arrow-left js-nav-prev"></a>
   <div class="toggle-nav-collapse hidden-xs">
-    <button href="#" class="js-toggle-nav" role="button">
+    <button class="js-toggle-nav" role="button">
       {{ macro.icon('chevron-left') }}
       <span class="sr-only js-nav-screen-text"></span>
```

Comment:
Shouldn't there be a text inside the span? Or is the text inserted with javascript?

Reply:
Yes the text will be inserted with javascript